packages/test/local-server-tests/src/test/documentDirty.spec.ts

```diff
@@ -319,9 +319,6 @@ describe("Document Dirty", () => {
 
                     checkDirtyState("after batch value set", true, 0);
 
-                    // Manually flush the ops so that they are sent as a batch.
```

Comment:
these should probably be changed to ordersequentially, I'm not sure if 'immediate' as flushmode would work